pyartcd/tests/pipelines/test_golang_builder_shipment.py

```
@@ -0,0 +1,345 @@
+        runtime.config = {}
+        runtime.working_dir = Path("/tmp/test-working")
+        pipeline = GolangBuilderShipmentPipeline(
+            runtime=runtime,
+            ocp_version="4.22",
+            golang_group="rhel-9-golang-1.25",
+            nvrs=["some-nvr"],
+            art_jira="ART-20930",
+        )
+        pipeline.env = "prod"
+        pipeline.release_plan = "ocp-art-golang-builder-prod-rhel9"
+        pipeline._gitlab_token = "test-token"
```

Ruff (0.15.17)

[error] Possible hardcoded password assigned to: "_gitlab_token"

(S105)